examples/complete/terraform.tfvars.tmp

```diff
@@ -0,0 +1,8 @@
+enabled_rules = ["cis"]
```

Comment:
This file should be removed but you may want to integrate this into the examples/complete test

Reply:
Removed!